app/brief_utils.py

```diff
@@ -13,6 +13,14 @@ def validate_brief_data(brief, enforce_required=True, required_fields=None):
         required_fields=required_fields
     )
 
+    criteria_weighting_keys = ['technicalWeighting', 'culturalWeighting', 'priceWeighting']
+    # Only check total if all weightings are set
+    if all(key in brief.data for key in criteria_weighting_keys):
+        criteria_weightings = sum(brief.data[key] for key in criteria_weighting_keys)
+        if criteria_weightings != 100:
+            for key in set(criteria_weighting_keys) - set(errs):
+                errs[key] = 'total_should_be_100'
```

Comment:
Fair enough. That does raise the question of why, if this is closer to brief responses, is it not changed to be implemented more like brief responses? ie. In `Brief.validate`